doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -37,60 +13,74 @@ The Cloud Infrastructure layer includes the physical infrastructure which is the
 The VIM is the OpenStack Wallaby release.
 
 This chapter is organised as follows:
-*	Consumable Infrastructure Resources and Services: these are infrastructure services and resources being exposed northbound for consumption
-    - Multi-tenancy with quotas
-      -	Virtual compute: vCPU / vRAM
-      -	Virtual storage: Ephemeral, Persistent and Image
-      -	Virtual networking – neutron standalone: network plugin, virtual switch, accelerator features
-      -	Virtual networking – 3rd party SDN solution
-      -	Additional network services: Firewall, DC Gateway
-* Cloud Infrastructure Management Software (VIM): is how we manage the Consumable Infrastructure Resources and Services
-    - VIM Core services (keystone, cinder, nova, neutron etc.)
+
+- Consumable Infrastructure Resources and Services: these are infrastructure services and resources being exposed northbound for consumption
+
+  - Multi-tenancy with quotas
+
+    - Virtual compute: vCPU / vRAM
+    - Virtual storage: Ephemeral, Persistent and Image
+    - Virtual networking – neutron standalone: network plugin, virtual switch, accelerator features
+    - Virtual networking – 3rd party SDN solution
+    - Additional network services: Firewall, DC Gateway
+
+- Cloud Infrastructure Management Software (VIM): is how we manage the Consumable Infrastructure Resources and Services
+
+  - VIM Core services (keystone, cinder, nova, neutron etc.)
+
     - Tenant Separation
     - Host aggregates providing resource pooling
     - Flavor* management
-*	Underlying Resources: are what provides the resources that allow the Consumable Infrastructure Resources and Services to be created and managed by the Cloud Infrastructure Management Software (VIM).
-    - Virtualisation
-    - Physical infrastructure
-      -	Compute
-      -	Network: Spine/Leaf; East/West and North/South traffic
-      -	Storage
+
+- Underlying Resources: are what provides the resources that allow the Consumable Infrastructure Resources and Services to be created and managed by the Cloud Infrastructure Management Software (VIM).
+
```

Comment:
I think this blank line is useless.

Reply:
@collivier 

For the sublist, the documentation https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#bullet-lists  says:

- This is a sublist.  The bullet lines up with the left edge of
    the text blocks above.  A sublist is a new list so requires a
    blank line above and below.